src/main/kotlin/no/nav/syfo/sykepengedagerinformasjon/kafka/SykepengedagerInformasjonKafkaConfig.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.syfo.sykepengedagerinformasjon.kafka
+
+import no.nav.syfo.config.kafka.KafkaConfig
+import org.apache.kafka.clients.consumer.ConsumerConfig
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.kafka.annotation.EnableKafka
+import org.springframework.kafka.config.ConcurrentKafkaListenerContainerFactory
+import org.springframework.kafka.core.ConsumerFactory
+import org.springframework.kafka.core.DefaultKafkaConsumerFactory
+import org.springframework.kafka.listener.ContainerProperties
+
+@EnableKafka
+@Configuration
+class SykepengedagerInformasjonKafkaConfig(
+    private val kafkaConfig: KafkaConfig,
+) {
+    @Bean
+    fun sykepengeDagerConsumerFactory(): ConsumerFactory<String, String?> {
+        val config = kafkaConfig.commonKafkaAivenConsumerConfig().toMutableMap().apply {
+            put(ConsumerConfig.GROUP_ID_CONFIG, "meroppfolging-backend-sykepengedager-01")
+            put(ConsumerConfig.MAX_POLL_RECORDS_CONFIG, "100")
+        }
+
+        return DefaultKafkaConsumerFactory(config)
+    }
+
+    @Bean
+    fun sykepengedagerInformasjonListenerContainerFactory(): ConcurrentKafkaListenerContainerFactory<String, String?> {
+        val factory = ConcurrentKafkaListenerContainerFactory<String, String?>()
+        factory.consumerFactory = sykepengeDagerConsumerFactory()
+        factory.containerProperties.ackMode = ContainerProperties.AckMode.MANUAL
```

Comment:
Jeg er litt usikker, men må man her ha factory.isBatchListener = true når poll records er over 1?

Jeg hadde ihvertfall det på sykmeldingfactory. Kanskje sjekke ut hva som blir mest riktig her.

Reply:
...om det funker og blir riktig i test så er det sikkert ikke så farlig

Reply:
Jeg har faktisk fjernet det etter at det [feilet i dev](https://logs.adeo.no/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:60000),time:(from:now-14d,to:now))&_a=(columns:!(level,message,envclass,application,pod),dataSource:(dataViewId:'96e648c0-980a-11e9-830a-e17bbd64b4db',type:dataView),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'96e648c0-980a-11e9-830a-e17bbd64b4db',key:envclass,negate:!f,params:(query:q),type:phrase),query:(match_phrase:(envclass:q))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:'96e648c0-980a-11e9-830a-e17bbd64b4db',key:application,negate:!f,params:(query:meroppfolging-backend),type:phrase),query:(match_phrase:(application:meroppfolging-backend)))),hideChart:!f,interval:auto,query:(language:lucene,query:'%22Method%20%5Bpublic%20void%20no.nav.syfo.sykepengedagerinformasjon.SykepengedagerInformasjonConsumer.listen(org.apache.kafka.clients.consumer.ConsumerRecord%3Cjava.lang.String,%20java.lang.String%3E,org.springframework.kafka.support.Acknowledgment)%5D%22'),rowsPerPage:500,sort:!(!('@timestamp',desc)))), @AudunSorheim